src/smbc.rs

```diff
@@ -61,6 +62,16 @@ fn is_einval<T: Eq + From<i8>>(t: T) -> IoResult<T> {
     }
 }
 
+/// Converts a pointer to the beginning of a C string and it's length and puts its
+/// equivelant value casted to u8
+fn str_ptr_to_u8(str_len: u32, str_ptr: *const i8, out_buffer: &mut Vec<u8>) {
+    for index in 0..str_len {
+        trace!(target: "smbc", "namelen : {}", str_len);
```

Comment:
Are you sure you want to print this on every iteration of the loop? I feel like it should be printed once beforehand